ciao-controller/controller_test.go

```diff
@@ -1150,6 +1209,12 @@ func TestGetStorageForVolume(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	//wl := types.Workload{
+	//	ID:      "validID",
+	//	ImageID: uuid.Generate().String(),
+	//	Storage: &s,
+	//}
+
```

Comment:
no - I had found that code in there and thought I didn't need it, so I commented it out to see if tests still worked without it, then forgot about it.